sampledb/config.py

```diff
@@ -123,6 +130,7 @@ def parse_configuration_values() -> None:
         'DISABLE_TOPICS',
         'ENABLE_FEDERATED_LOGIN',
         'ENABLE_FEDERATED_LOGIN_CREATE_NEW_USER',
+        'OIDC_ONLY',
```

Comment:
```suggestion
        'OIDC_ONLY',
        'OIDC_DISABLE_NONCE',
```

Hopefully this'll be the final small change :)

Reply:
... because all other final changes will require a major rewrite? ;)

Done.